components/sensor/voltageMeasure.c

```diff
@@ -1,23 +1,23 @@
-// Copyright 2022 PWrInSpace
+// // Copyright 2022 PWrInSpace
 
 #include "voltageMeasure.h"
 
-bool voltageMeasureInit(voltageMeasure *vMes) {
-  // vMes->channel = channel;
-  adc1_config_width(ADC_WIDTH_BIT_DEFAULT);
-  adc1_config_channel_atten(vMes->channel, ADC_ATTEN_DB_11);
+// bool voltageMeasureInit(voltageMeasure *vMes) {
```

Comment:
dlaczego wywalone?

Reply:
to do zrobienia na innym branchu - skutki przejścia na idf 5.0